### PR TITLE
PAYARA-4002 Replace org.glassfish.j2ee.statistics with managment-api

### PR DESCRIPTION
**Pull request by Cousjava**

### CODE REVIEW

**Comment by Cousjava** [comment]
Jenkins test please

**Comment by Cousjava** [comment]
Jenkins test please

File: appserver/common/stats77/pom.xml [hunk 1]
```diff
@@ -72,5 +73,11 @@
             <artifactId>internal-api</artifactId>
             <version>${project.version}</version>
         </dependency>
+        <dependency>
+            <groupId>javax.management.j2ee</groupId>
+            <artifactId>javax.management.j2ee-api</artifactId>
+            <version>${javax.management.j2ee-api.version}</version>
```
Comment (pdudits) [inline appserver/common/stats77/pom.xml]: Did you consider
```
    <groupId>jakarta.management.j2ee</groupId>
    <artifactId>jakarta.management.j2ee-api</artifactId>
```

Comment (Cousjava) [reply appserver/common/stats77/pom.xml]: javax.management is already in Payara, switching to the Jakarta version is a separate issue.

Comment (pzygielo) [reply appserver/common/stats77/pom.xml]: As it is managed in https://github.com/payara/Payara/blob/13bd97b3cd8a0c9d3c51fe24eef40f50f93d03b4/appserver/pom.xml#L1006-L1010 could the version element be skipped here then?

**Comment by Cousjava** [comment]
Jenkins test please

**Pull request merged**